web/src/lib/components/pages/record.tsx

```diff
@@ -268,12 +263,11 @@ export default class RecordPage extends Component<RecordProps, RecordState> {
     this.props.onRecordStop && this.props.onRecordStop();
   }
 
-  newSentenceSet() {
-    // If we don't have any sentences in our cache, fill it and try again.
-    if (this.sentenceCache.length < SET_COUNT) {
+  private async newSentenceSet() {
+    // if we don't have enough sentences in our cache, fill it before continuing
```

Comment:
I've learned to prefer full proper sentences as comments (ie. capitalize first letter, and period). I picked that up from Firefox devs. Comments are slightly easier to read when a proper sentence.

Reply:
👍